inference-engine/src/mkldnn_plugin/nodes/mkldnn_interpolate_node.cpp

```diff
@@ -386,6 +386,13 @@ struct jit_uni_interpolate_kernel_f32 : public jit_uni_interpolate_kernel, publi
         Xbyak::Label out_loop_label;
         Xbyak::Label out_loop_end;
 
+        Xbyak::Reg64 reg_work_amount_bk = reg_src_aux2;
```

Comment:
Not sure I got the fix. I see that you are using [rcx](https://github.com/openvinotoolkit/openvino/blob/2ecd24e7ac85574ff8434efa865cf4dd659550bd/inference-engine/src/mkldnn_plugin/nodes/mkldnn_interpolate_node.cpp#L182) register which is [reserved](https://github.com/openvinotoolkit/oneDNN/blob/0813c00df7558bc9b858d3a73c725bab2ce1b1eb/src/cpu/x64/jit_generator.hpp#L100) for param1 ptr on windows by ABI rules. So changing the register for reg_d_bias  should help to solve the problem. Or I missing smt?

Reply:
@chenhu-wang  Thanks for explanation. I think it is deserved only in case it will improve code readability. Otherwise - lets keep as is.